src/govuk/components/header/_index.scss

```diff
@@ -207,9 +207,21 @@
       margin-left: govuk-spacing(1);
     }
 
+    &[aria-expanded="true"]:after {
```

Comment:
Does this mean we can get rid of the `--open` modifier class?

Reply:
Whoops! Yes it does. Cheers for flagging.